src/Security/InteractiveSecurityHelper.php

```diff
@@ -240,4 +240,23 @@ public function guessEmailGetter(SymfonyStyle $io, string $userClass): string
             $classMethods
         );
     }
+
+    public function guessIdGetter(SymfonyStyle $io, string $userClass): string
+    {
+        $reflectionClass = new \ReflectionClass($userClass);
+
+        if ($reflectionClass->hasMethod('getId')) {
+            return 'getId';
+        }
+
+        $classMethods = [];
+        foreach ($reflectionClass->getMethods() as $method) {
+            $classMethods[] = $method->name;
+        }
```

Comment:
I guess we could refactor this into a private method since we do it now at least twice. And it occurred to me that this should only include methods that are public.

Reply:
done